src/opm/input/eclipse/Schedule/Network/ExtNetwork.cpp

```diff
@@ -113,15 +154,15 @@ std::optional<Branch> ExtNetwork::uptree_branch(const std::string& node) const {
     if (branches.size() == 1)
         return std::move(branches[0]);
 
-    throw std::logic_error("Bug - more than upstree branch for node: " + node);
+    throw std::logic_error("Bug - more than uptree branch for node: " + node);
```

Comment:
probably should be more than **one** uptree branch for node

Reply:
Ah, yes.. :)